docs/COMMIT.md

```diff
@@ -4,7 +4,7 @@ PIConGPU Commit Rulez
 We agree on the following simple rules to make our lives easier :)
 
 - Stick to the **style** below for **commit messages**
-- **Commit compiling patches** for the *main* branches (`master` and `dev`),
+- **Commit compiling patches** for the `dev` branche,
```

Comment:
please remove the 'e' at the end of 'branche'.

Reply:
great catch - fixed